e2e/tests/components/select/standard/select.e2e-spec.ts

```diff
@@ -564,7 +564,7 @@ describe('Select Tests', () => {
         await page.clickOnDropdown(false);
 
         // Initial state: recent option list is not shown
-        // expect(await imageCompare('select-open-single')).toEqual(0);
+        expect(await imageCompare('select-open-single')).toEqual(0);
```

Comment:
Were these tests previously commented out as part of the Angular 9 changes? Should this change be included?

Reply:
We commented out all of the screenshot tests in the select compoennt about a month back because they were failing regularly. Tara put in some fixes to make them more stable, so the image checks were uncommented at that point.

Reply:
Grand, just watched to check. I have pushed a fix for another service I noticed was failing, once that builds then this should be good to merge if all tests pass etc..

Reply:
That's great, thank you for spotting that one.